typescript/forms.d.ts

```diff
@@ -1132,7 +1132,54 @@ export type BuiltInFormElementLookup = NewBuiltInFormElementLookup & {
 }
 
 export type FormElementLookupEnvironmentUrl = FormElementEnvironmentUrl
-export type NewFormElementLookup = {
+
+export type FormElementLookupTableRowBase = {
+  /** value entered into the element used to trigger the lookup */
+  inputValue: string
+}
+
+export type FormElementTableRowText = FormElementLookupTableRowBase & {
+  /** the value to populate the configured element with */
+  preFillValue: string
+}
+
+export type FormElementTableRowNumber = FormElementLookupTableRowBase & {
+  /** The value to prefil the correpsonding column elementName with */
+  preFillValue: number
+}
+
+export type FormElementLookupTableColumnBase = {
+  /** the name of the form element to prefill */
+  elementName: string
+}
+
+export type FormElementLookupTableColumnText = {
```

Comment:
should this be using the `FormElementLookupTableColumnBase` type above?

Reply:
Yep, nice catch